components/demo/demo-snippet.js

```diff
@@ -203,6 +217,25 @@ class DemoSnippet extends LitElement {
 		this._code = textNode.textContent;
 	}
 
+	_updateHasSkeleton() {
+
```

Comment:
This code is auto-detecting whether the component in the demo snippet is skeleton-aware and if so, it shows the toggle.